src/smart-app-banner.scss

```diff
@@ -0,0 +1,319 @@
+//colors
```

Comment:
for `scss` files there's an easier way of linting files and generating better `css` files from them, but we can skipp that and remove it